### PR TITLE
GOB: Distinguish Adibou 2 versions after version numbers

### PR DESCRIPTION
**Pull request by BJNFNE**

### CODE REVIEW

File: engines/gob/detection/tables_adibou2.h [hunk 1]
```diff
@@ -72,7 +72,8 @@
 	{
 		"adibou2",
 		"ADIBOU 2.10 Environnement",
-		AD_ENTRY1s("intro.stk", "f2b797819aeedee557e904b0b5ccd82e", 8736454),
+		AD_ENTRY2s("intro.stk", "f2b797819aeedee557e904b0b5ccd82e", 8736454,
+				"BECBF210.CD1",	"bc828c320908a5eaa349956d396bd8e1", 8),
```
Comment (bluegr) [inline engines/gob/detection/tables_adibou2.h:76]: I don't understand, why is this change needed, since intro.stk has a unique MD5 hash?

File: engines/gob/detection/tables_adibou2.h [hunk 2]
```diff
@@ -86,7 +87,8 @@
 	{
 		"adibou2",
 		"ADIBOU 2.11 Environnement",
-		AD_ENTRY1s("intro.stk", "7b1f1f6f6477f54401e95d913f75e333", 8736904),
+		AD_ENTRY2s("intro.stk", "7b1f1f6f6477f54401e95d913f75e333", 8736904,
+				"BECBF211.CD1",	"bc828c320908a5eaa349956d396bd8e1", 8),
```
Comment (bluegr) [inline engines/gob/detection/tables_adibou2.h:91]: Same here, intro.stk already has a unique MD5 hash

File: engines/gob/detection/tables_adibou2.h [hunk 3]
```diff
@@ -100,7 +102,8 @@
 	{
 		"adibou2",
 		"ADIBOU 2.12 Environnement",
-		AD_ENTRY1s("intro.stk", "1e49c39a4a3ce6032a84b712539c2d63", 8738134),
+		AD_ENTRY2s("intro.stk", "1e49c39a4a3ce6032a84b712539c2d63", 8738134,
+				"BECBF212.CD1",	"bc828c320908a5eaa349956d396bd8e1", 8),
```
Comment (bluegr) [inline engines/gob/detection/tables_adibou2.h:106]: This one too, it's a unique hash, why add another file?

File: engines/gob/detection/tables_adibou2.h [hunk 4]
```diff
@@ -114,7 +117,8 @@
 	{
 		"adibou2",
 		"ADIBOU 2.13s Environnement",
-		AD_ENTRY1s("intro.stk", "092707829555f27706920e4cacf1fada", 8737958),
+		AD_ENTRY2s("intro.stk", "092707829555f27706920e4cacf1fada", 8737958,
+				"BECBD213.CD1",	"bc828c320908a5eaa349956d396bd8e1", 8),
```
Comment (bluegr) [inline engines/gob/detection/tables_adibou2.h:121]: This one is correct, since the MD5 hash of intro.stk is not unique, so adding another file for correct detection is the correct approach

**Comment by BJNFNE** [comment]
i did this changes for the reason, that if you add these one of Adibou2 which are identical so if you add an Game to ScummVM via "Add Game" it give you the option 2.13 2.13s and 2.20, so it only shows with these changes the version that you own.

**Comment by bluegr** [comment]
Ah, that makes sense. Thanks for your work!

File: engines/gob/detection/tables_adibou2.h (hunk 4 above)
Comment (tag2015) [reply engines/gob/detection/tables_adibou2.h:121]: @BJNFNE please check this entry. Since it's the french version the .CD1 file should be BECBF213.CD1

File: engines/gob/detection/tables_adibou2.h [hunk 5]
```diff
@@ -206,7 +212,8 @@
 	{
 		"adibou2",
 		"ADIBÙ 2.13 Ambiente",
-		AD_ENTRY1s("intro.stk", "092707829555f27706920e4cacf1fada", 8737958),
+		AD_ENTRY2s("intro.stk", "092707829555f27706920e4cacf1fada", 8737958,
+				"BECBD213.CD1",	"bc828c320908a5eaa349956d396bd8e1", 8),
```
Comment (tag2015) [inline engines/gob/detection/tables_adibou2.h:216]: Same here. The italian version should be BECBI213.CD1

Comment (BJNFNE) [reply engines/gob/detection/tables_adibou2.h:216]: can you provide me the md5 hash and filesize or is it the same?

File: engines/gob/detection/tables_adibou2.h (hunk 4 above)
Comment (BJNFNE) [reply engines/gob/detection/tables_adibou2.h:121]: can you provide me the md5 hash and filesize or is it the same?

Comment (tag2015) [reply engines/gob/detection/tables_adibou2.h:121]: > can you provide me the md5 hash and filesize or is it the same?

You changed the entry so I thought you had the game? Anyway the hash/filesize is the same (which is not a great thing, it would have been better to use a more meaningful file instead of this one which is identical in all releases and just contains a string)

File: engines/gob/detection/tables_adibou2.h (hunk 5 above)
Comment (tag2015) [reply engines/gob/detection/tables_adibou2.h:216]: Same here too
